test/dynamo/test_optimizers.py

```diff
@@ -45,10 +45,8 @@ def fn():
 
 def make_test(optim_cls, closure=None, **kwargs):
     opt = optim_cls(model.parameters(), **kwargs)
-
```

Comment:
Super minor nit but could you add back these spaces so we aren't touching code unnecessarily?